lua/tardis/settings/sh_global_settings.lua

```diff
@@ -44,6 +44,22 @@ TARDIS:AddSetting({
     name="MaxHealth",
 })
 
+TARDIS:AddSetting({
+    id="long_repair",
+    type="bool",
+    value=true,
+
+    class="global",
+    convar = {
+        name = "tardis2_long_repair",
+        flags = CONVAR_FLAGS,
+        desc = "TARDIS - Whether repairing process lasts 60 seconds",
```

Comment:
This should clarify that it is 'up to' 60 seconds, and also that having it off will make it 'up to' 10 seconds

Additionally, this should be using the language features